sdk/src/types/block/output/metadata.rs

```diff
@@ -126,7 +126,7 @@ mod dto {
         fn try_from(value: OutputMetadataDto) -> Result<Self, Self::Error> {
             Ok(Self {
                 block_id: value.block_id,
-                output_id: OutputId::new(value.transaction_id, value.output_index)?,
+                output_id: OutputId::new(value.transaction_id, value.output_index),
```

Comment:
This was the only thing that returned a result, want to impl From instead now?

Reply:
Good catch

Reply:
Done